src/node_crypto.cc

```diff
@@ -482,12 +485,13 @@ void SecureContext::SetKey(const FunctionCallbackInfo<Value>& args) {
   if (!bio)
     return;
 
-  node::Utf8Value passphrase(env->isolate(), args[1]);
-
+  auto callback = has_password ? PasswordCallback : NoPasswordCallback;
```

Comment:
Fair point. Let me restore the original allowing a null passphrase but keep the NoPasswordCallback for this.